mapper-processor/src/main/java/com/datastax/oss/driver/internal/mapper/processor/entity/EntityHelperGenerator.java

```diff
@@ -119,6 +113,10 @@ public String addEntityHelperField(ClassName childEntityName) {
     GeneratedCodePatterns.addFinalFieldAndConstructorArgument(
         ClassName.get(MapperContext.class), "context", classContents, constructorContents);
 
+    // TODO create name converter
```

Comment:
Is that TODO still relevant? 
I think that we have name converter for entity name implemented, right?

Reply:
Good catch, it is indeed outdated.

I ended up inlining the retrieval of the converter everywhere instead of storing it in a field. For example in `NamingStrategyIT_NameConverterEntityHelper__MapperGenerated`:
```java
target = target.setInt(context.getNameConverter(NamingStrategyIT.TestNameConverter.class).toCassandraName("entityId"), entity.getEntityId());
```
This might seem a bit extreme, but `getNameConverter` is cached so there shouldn't be much of a performance hit.

Also TBH I don't want to spend to much time optimizing that case, because it should be the exception rather than the norm. If your naming strategy *for a single entity* can't be covered by a default convention (with maybe a few `@CqlName` here and there), it's probably a bit whacky.